src/main/java/com/example/demo/service/shoppingcart/ShoppingCartServiceImpl.java

```diff
@@ -25,10 +25,15 @@ public class ShoppingCartServiceImpl implements ShoppingCartService {
     private final ShoppingCartMapper cartMapper;
 
     @Override
-    public ShoppingCartDto getShoppingCartByUserId(Long id) {
+    public ShoppingCartDto getShoppingCartDtoByUserId(Long id) {
         return cartMapper.toDto(getCartByUserId(id));
     }
 
+    @Override
+    public ShoppingCart getShoppingCartByUserId(Long id) {
+        return getCartByUserId(id);
+    }
+
```

Comment:
why you need this method, use getCartByUserId directly, It is 2 identical methods but better to use getShoppingCartByUserId name